### PR TITLE
I1423, fixing style name problem on RTFD builds

### LINKED ISSUE #1423: fatal error in mswms gallery on RTFD build
seen this on the RTFD build of 6.0.6 

```
Configuration error:
There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/mss/conda/stable/lib/python3.9/site-packages/sphinx/config.py", line 332, in eval_config_file
    exec(code, namespace)
  File "/home/docs/checkouts/readthedocs.org/user_builds/mss/checkouts/stable/docs/conf.py", line 67, in <module>
    mslib.mswms.wms.server.generate_gallery(sphinx=True, generate_code=True, all_plots=True, levels="3,4,200,300",
  File "/home/docs/checkouts/readthedocs.org/user_builds/mss/checkouts/stable/docs/../mslib/mswms/wms.py", line 244, in generate_gallery
    self.__init__()
  File "/home/docs/checkouts/readthedocs.org/user_builds/mss/checkouts/stable/docs/../mslib/mswms/wms.py", line 202, in __init__
    self.register_hsec_layer(datasets, layer)
  File "/home/docs/checkouts/readthedocs.org/user_builds/mss/checkouts/stable/docs/../mslib/mswms/wms.py", line 456, in register_hsec_layer
    raise ValueError(f"new layer is already registered? dataset={dataset} layer.name={layer.name} "
ValueError: new layer is already registered? dataset=ecmwf_EUR_LL015 layer.name=BASEMAP new=<mslib.mswms.mpl_hsec.MPLBasemapHorizontalSectionStyle object at 0x7f6ffa2921c0> old=<mslib.mswms.mpl_hsec_styles.HS_GenericStyle object at 0x7f6ffa2a92e0>
```

This can locally reproduced by MSS/docs$ make html

### PR DESCRIPTION
**Pull request by ReimarBauer**

### CODE REVIEW

File: mslib/mswms/mpl_vsec_styles.py [hunk 1]
```diff
@@ -114,6 +114,7 @@ class VS_GenericStyle(AbstractVerticalSectionStyle):
     """
     Vertical section of chemical species/other stuff
     """
+    name = "VS_GenericStyle"
```
Comment (joernu76) [inline mslib/mswms/mpl_vsec_styles.py:117]: I do not understand the purpose of this.
This class should not be used per se, only classes derived from this, which get assigned there own name.

How does this solution connect to issue 1423?

The (new) assert should discover mis-configurations, where multiple identical layers are configured, which, previously, would choose the last one to be used.

The error should only be generated for faulty configuration...?

Comment (ReimarBauer) [reply mslib/mswms/mpl_vsec_styles.py:117]: hmm, propabably. The name helps to destinguish where "BASEMAP" comes from. I'll also add an action script for building the docs.

**Review by joernu76 — DISMISSED** [review]

File: mslib/mswms/mpl_vsec_styles.py (hunk 1 above)
Comment (ReimarBauer) [reply mslib/mswms/mpl_vsec_styles.py:117]: found the cause for this.

**Comment by ReimarBauer** [comment]
the new workflow errors not on traceback

**Review by joernu76 — APPROVED** [review]

**Pull request merged**